megamek/resources/megamek/common/templates/tro/aero.ftlh

```diff
@@ -71,6 +71,7 @@
 	<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Safe Thrust:</td><td align="center">${safeThrust}</td><td></td></tr>
 	<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Max Thrust:</td><td align="center">${maxThrust}</td><td></td></tr>
 	<tr><td>Structural Integrity:</td><td align="center">${si}</td><td></td></tr>
+	<tr><td>VSTOL Equipment:</td><td align="center"></td><td align="center">${vstolMass}</td></tr>
```

Comment:
This adds a line for VSTOL equipment on all aerospace units, but it's only relevant for conventional fighters.

Reply:
Okay, this is weird, I swear that was between two if statements with the VSTOL checks.... Will fix this now.